### DISCUSSION TITLE
How to customize reply when information is not found?

### DISCUSSION THREAD

**Discussion by myusoutharm** [microsoft/sample-app-aoai-chatGPT#951] [Q&A]
For the "bring your own data" use case, how to customize the default reply when no information is found? The default reply is "The information about x is not available in the retrieved documents. Please provide another query related to the content in the documents for assistance.". Need to customize it by adding a link to a form that users can submit/suggest documents for a question, and the attachment will be uploaded to Blob, which will be indexed daily.

Thanks.

**Comment by stbere** [comment]
When you get that reply, right-click and choose "Inspect" -- you should be able to find which page in your code references this part. I've done this before but can't remember exactly which file it was that made the edit.
